public/app.js

```diff
@@ -19,7 +19,8 @@ $(function() {
             method: 'POST',
             dataType: 'json',
             data: {
-                phoneNumber: $('#phoneNumber').val()
+                phoneNumber: $('#phoneNumber').val(),
+                salesNumber: $('#salesNumber').val()
```

Comment:
what do you think about this @acamino? I did notice the numbers come with a bunch of spaces, sometimes dashes. Do you think it's a good idea to clean them up before using them with twilio api? It works as is but don't know if this is bad practice.